starter_code/src/movies.js

```diff
@@ -2,14 +2,399 @@
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
 
+function orderByYear(itemsToSort) {
+  let sortedItems = [...itemsToSort];
+
+  sortedItems.sort(function(a, b) {
+    if (a.year === b.year) {
+      return a.title.localeCompare(b.title);
+    }
+    return a.year - b.year;
+  });
+  return sortedItems;
+}
+
+let unsortedMovies = [
```

Comment:
This is not necessary as all the movies is unsorted within the data file and since the data file has been linked in the html file you can console.log(movies) and you will see the whole array of unsorted movies